src/include/sof/platform.h

```diff
@@ -47,6 +47,28 @@ int platform_init(struct sof *sof);
 
 /** @}*/
 
+#define HOST_TO_LOCAL 0
+#define LOCAL_TO_HOST 1
+
+#ifdef CONFIG_IMX8ULP
+static inline void convert_addr(int type, uint32_t *addr)
+{
```

Comment:
This function doesn't belong to this commit. Current commit is about Messaging Unit.